src/lib/es2015.iterable.d.ts

```diff
@@ -203,15 +203,7 @@ interface PromiseConstructor {
      * @param values An array of Promises.
      * @returns A new Promise.
      */
-    all<TAll>(values: Iterable<TAll | PromiseLike<TAll>>): Promise<TAll[]>;
-
-    /**
-     * Creates a Promise that is resolved or rejected when any of the provided Promises are resolved
-     * or rejected.
-     * @param values An array of Promises.
-     * @returns A new Promise.
-     */
-    race<T>(values: Iterable<T | PromiseLike<T>>): Promise<T>;

```

Comment:
Why was `race` removed?

Reply:
48419846c71073cd4b4e75ccae465d033b2ac97c it was [moved](https://github.com/microsoft/TypeScript/pull/31117/files#diff-8b82636b72c8b4bf2374e47bc4a90bdfR109) to es2015.promise.d.ts in #31117

Reply:
It existed here so that you could include `es2015.promise` without `es2015.iterable`, depending on what shims/polyfills are used down-level, although it looks like a definition including `Iterable` was incorrectly added to that file at some point.

Reply:
I'm happy to fix that but just wondering, is it better to be able to include `es2015.promise` without `es2015.iterable`, or `es2015.iterable` without `es2015.promise`? The latter is more obvious (all `Promise.all()` and `Promise.race()` definitions located together) and has fewer overloads (since `readonly T[]` extends `Iterable<T>`).

Reply:
Sorry, I get it now: `es2015.iterable` doesn't depend on `PromiseConstructor`, it *augments* it. I'll move that `Iterable` overload back to `es2015.iterable`.

Reply:
Done.